tests/integration/test_bayesian_optimization.py

```diff
@@ -68,7 +68,7 @@
             (20, EfficientGlobalOptimization()),
             (25, EfficientGlobalOptimization(AugmentedExpectedImprovement().using(OBJECTIVE))),
             (
-                15,
+                25,
```

Comment:
I think it also works with this:

```suggestion
                21,
```

Reply:
It only works with 22 on my machine.

Reply:
updated to 22 as the min known to fix all local failures